src/components/molecules/postIcons/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { FaRegComment } from "react-icons/fa"
+import { AiOutlineRetweet } from "react-icons/ai"
+import { AiOutlineHeart } from "react-icons/ai"
+import { GiHistogram } from "react-icons/gi"
+import { LuShare } from "react-icons/lu"
+import AtomsRoundButton from "~/components/atoms/button/round-button";
+
+const MoleculesPostIcons = () => (
```

Comment:
```suggestion
const MoleculesPostIcons = () => {
  const postIconsArray = [
    {
      icon: FaRegComment,
      buttonStyle: 'hover:text-sky-500 hover:bg-sky-100/20'
    },
    {
      icon: AiOutlineRetweet,
      buttonStyle: 'hover:text-green-500 hover:bg-green-100/20'
    },
    {
      icon: AiOutlineHeart,
      buttonStyle: 'hover:text-red-500 hover:bg-red-100/20'
    },
  ]
  return (
    <div className='text-sm mt-3 flex justify-between flex-1'>
      {
        postIconsArray.map( ({ icon: Icon, buttonStyle }) => (
          <AtomsRoundButton className={buttonStyle}>
            <Icon />
          </AtomsRoundButton>
        ))
      }
    </div>
  )
  
}

```
これはあくまで、共有です！
今後例えばたくさんアイコンが追加される場合や、こちらのMoleculesを
編集したい場合には管理しやすいように重複しているコードはなるべく取り除く方が
Maintenanceがしやすくなります！

Reply:
これ良いーーーー！

Reply:
かっこいいコード書いていきましょ💪